contrib/kind-helm.sh

```
@@ -512,6 +522,10 @@ print_params() {
      echo "OVN_LOG_LEVEL_SB = $OVN_LOG_LEVEL_SB"
      echo "OVN_LOG_LEVEL_NORTHD = $OVN_LOG_LEVEL_NORTHD"
      echo "OVN_LOG_LEVEL_CONTROLLER = $OVN_LOG_LEVEL_CONTROLLER"
+     echo "ENABLE_TRACING = $ENABLE_TRACING"
+     echo "TRACING_NAMESPACE = $TRACING_NAMESPACE"
+     if [[ "$ENABLE_TRACING" == true ]]; then
```

Shellcheck (0.11.0)

[error] Couldn't find 'fi' for this 'if'.

(SC1046)